lib/agent0/agent0/interactive_fuzz/fuzz_long_short_maturity_values.py

```diff
@@ -28,17 +27,38 @@ def main(argv: Sequence[str] | None = None):
     """
     # Setup the experiment
     parsed_args = parse_arguments(argv)
+    fuzz_long_short_maturity_values(*parsed_args)
+
+
+def fuzz_long_short_maturity_values(num_trades: int, chain_config: LocalChain.Config | None = None):
+    """Does fuzzy invariant checks on closing longs and shorts past maturity.
+
+    Parameters
+    ----------
+    num_trades: int
+        Number of trades to perform during the fuzz tests.
+    chain_config: LocalChain.Config, optional
+        Configuration options for the local chain.
+
+    Raises
+    ------
+    AssertionError
+        If the invariant checks fail during the tests an error will be raised.
+    """
+
     log_filename = ".logging/fuzz_long_short_maturity_values.log"
     # Parameters for local chain initialization, defines defaults in constructor
     # set a large block time so i can manually control when it ticks
     # TODO: set block time really high after contracts deployed:
     # chain_config = LocalChain.Config(block_time=1_000_000)
     # TODO: We will want to log the random seed; so remove this pylint disable once we do
-    chain, random_seed, rng, interactive_hyperdrive = setup_fuzz(log_filename)  # pylint: disable=unused-variable
+    chain, _, rng, interactive_hyperdrive = setup_fuzz(
```

Comment:
I left this unused variable here because we need to log it in a follow-up PR (today). The TODO above this line and the pylint disable on this line are also related to this.

Reply:
sheng logs it in https://github.com/delvtech/agent0/pull/1177

Reply:
I told him to delete the text here: https://github.com/delvtech/agent0/pull/1177/files#r1419733587

Reply:
easy to put back when u need it